imdb-keras/Dockerfile

```diff
@@ -2,4 +2,5 @@ FROM python:3.8.9
 COPY requirements.txt /app/
 WORKDIR /app
 RUN pip install -r requirements.txt
-RUN pip install -e git+https://github.com/scaleoutsystems/fedn.git#egg=fedn\&subdirectory=fedn
+RUN pip install -e git+https://github.com/scaleoutsystems/fedn.git@develop#egg=fedn\&subdirectory=fedn
```

Comment:
Why the need to change the install to develop instead of main?

Reply:
I agree. I can't test fedn on my own machine anyway because of this issue:
https://github.com/scaleoutsystems/fedn/issues/393